### DISCUSSION TITLE
Lighthouse deck does not receive data

### DISCUSSION THREAD

**Discussion by David-Glom** [bitcraze/discussions#1327] [Q&A]
Hi,
on one of our lighthouse deck it is impossible receive signals. Using the same drone with an other deck works perfectly fine, so the problem has to be on the lighthouse deck. In the consol everything seems right, so that means there is no problem with one specific receiver, right? 
On the deck there are two leds on, a blue on and a orange on. On the drone, the M2 and M3 led are constantly on, the M1 LED is blinking and the M4 led is shining green( because I'm connected to the drone). 
This is the console output:
![correct console output](https://github.com/bitcraze/discussions/assets/129954836/67cfacdd-0ec5-4ae5-8e43-3d526d1fcee1)
I know, the information on this problem is not much, but I don't know, how to offer more useful information :/

Thanks in advance!

**Comment by knmcguire** [comment]
Just a quick question in between, was this the same deck as mentioned here? https://github.com/orgs/bitcraze/discussions/1326

**Reply by David-Glom** [reply]
No this are two different decks, so I thought it would be the best to have two different discussions to divide the problems, although both belong to the lighthouse deck. We have also around 5 working drones with lighthouse decks, so the general setup works.

**Reply by tobbeanton** [reply]
It seems the LH-deck does not receive any data, if so the green or orange LED should blink. You can find more details on page 8 in [this](https://www.bitcraze.io/about/events/documents/bam2021/Advanced_Lighthouse_Usage_bam2021.pdf) presentation.

How does the sensors look on the deck? Perhaps you could share a high resolution photo of the deck?

**Reply by knmcguire** [reply]
@David-Glom the link to the presentation pdfs is https://www.bitcraze.io/about/events/documents/bam2021/Advanced_Lighthouse_Usage_bam2021.pdf

**Reply by David-Glom** [reply]
Hi, thanks for your answer :smile: Here is a high resolution photo of the deck:
![Lighthouse deck](https://github.com/bitcraze/discussions/assets/129954836/99218189-6428-42fd-8194-350c6aa14581)
If you want to see some of the parts more closely, for example through a microscope, just write it :) 

If I open the cfclient, the drone starts in the original zero point in the co-ordination system, but then drifts away, which you can see in this image:
![cfclient lighthouse tab](https://github.com/bitcraze/discussions/assets/129954836/cb323dd5-e781-499a-b78d-4ad92bb7646a)

Hope this information can help finding out my problem :)

**Reply by knmcguire** [reply]
So, is the orange LED is solid right? not blinking?

**Reply by David-Glom** [reply]
Yes it is solid.

**Reply by tobbeanton** [reply]
It feels like there is some HW problem going on that will be hard for you to debug. Can you please send us an email to support@bitcraze.io and we will arrange shipping and replacement(s).

**Reply by kleikemper** [reply]
I have the same problem with one of our Lighthouse decks

**Comment by David-Glom** [comment]
> @David-Glom the link to the presentation pdfs is https://www.bitcraze.io/about/events/documents/bam2021/Advanced_Lighthouse_Usage_bam2021.pdf
Thanks for sharing the right link, I somehow landed on the google docs link, but I can now open the presentation :smiley:
